### ISSUE TITLE
HTML element does not have a lang attribute

### ISSUE THREAD

**Issue by tejasv23** [CMU-313/Teedy#119]

**Comment by tejasv23** [comment]
I would like to work on this please!

**Comment by tejasv23** [comment]
The current score for accessibility I observe is 86.
The warning that I get is the following: 

If a page doesn't specify a lang attribute, a screen reader assumes that the page is in the default language that the user chose when setting up the screen reader. If the page isn't actually in the default language, then the screen reader might not announce the page's text correctly.
